app/views/active_admin/_flash_messages.html.erb

```diff
@@ -0,0 +1,22 @@
+<% if flash_messages.present? %>
```

Comment:
It's an interesting decision by ActiveAdmin 4 to copy a bunch of these partials into application codebases. I can see some upsides and downsides. We'll see how it goes. I guess I don't have much choice, anyway, though, if I want to stay on the latest version of ActiveAdmin. It doesn't seem terrible, so far, at least.

There were a number of other templates injected initially that I deleted before first pushing this PR. Most of the templates that I deleted were Devise web views or mailers. I don't think that I use that ActiveAdmin functionality.